oncogene.js

```diff
@@ -56,6 +56,10 @@ class Oncogene {
 
     setVal(key, selectedVarient) {
         const step = this.getStep()
+        if(selectedVarient.skip === true) {
```

Comment:
I personally prefer explicit checks but if you don't I will change

Explicit values helps in testing and prevents user mistakes